packages/builder/src/components/design/settings/controls/DraggableList/DraggableList.svelte

```diff
@@ -39,14 +40,20 @@
   let draggableItems = []
 
   const buildDraggable = items => {
-    return items
+    const draggableItems = items
       .map(item => {
         return {
           id: listItemKey ? item[listItemKey] : generate(),
           item,
         }
       })
       .filter(item => item.id)
+
+    if (whitelist) {
```

Comment:
Remove the whitelist behaviour from within the core DraggableList component and replicate the behaviour in a component dedicated to configuring columns.